README.md

```diff
@@ -136,6 +136,43 @@ rpc:
     batchDelay: 100
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +58 to +61]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->do I understand correctly that for `rpc.blockReceipts.enabled` even if this conf is completely missing from both `3. Configuration files` and `2. Environment variables`, it will still be [set](https://app.graphite.dev/github/pr/thirdweb-dev/insight/102/Add-support-for-eth_getBlockReceipts-RPC-method#file-cmd/root.go:L45) to `true` because `1. Command line arguments` has it defaulting to `true` even if `rpc-blockReceipts-enabled` flag is missing?

(just validating my overall understanding of config setup with this good example)<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/insight/102?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
correct